projects/storefrontapp/src/app/spartacus/features/opf/opf-feature.module.ts

```diff
@@ -33,6 +34,8 @@ if (environment.b2b) {
         baseUrl:
           'https://opf-dev.api.commerce.stage.context.cloud.sap/commerce-cloud-adapter/storefront',
         commerceCloudPublicKey: 'ab4RhYGZ+w5B0SALMPOPlepWk/kmDQjTy2FU5hrQoFg=',
+        successUrl: 'redirect/success',
```

Comment:
I would prefer to pass the absolute URL, not only a route. Maybe we can create a service and method which will return an absolute url based on routing service. For example: `getAbsoluteUrl('redirect/success')`. As an argument we can pass a route path.

Reply:
see previous comments about fetching current domain url before components load. can we access window.location at this stage?